### PR TITLE
As a user of the WWW site I want the News/Events widget to be accessible.

### PR DESCRIPTION
**Pull request by tatwell**
Trello Card: https://trello.com/c/UVkFhbQx

Issue: #252

### CODE REVIEW

File: .cascade-code/_cascade/formats/modular/widgets/Featured_News_Events_Left_Col [hunk 1]
```diff
@@ -1,97 +1,89 @@
...
-                    <div class="title"></div>
-                    <div class="description">View the published page to see this section.</div>
-                    <div class="readMore"></div>
-                </div>
-            </li>
-            <li>
-                <div class="news clearfix">
-                    <img class="loading" src="/_files/level/img/ajax-loader.gif" alt="Loading news list..."/>
-                    <div class="story story1" itemscope="itemscope" itemtype="http://schema.org/Article">
+#*
```
Comment (kaylaziegler) [inline .cascade-code/_cascade/formats/modular/widgets/Featured_News_Events_Left_Col]: Shouldn't these two velocity formats have the .vtl extension?

**Comment by mandythomas** [comment]
@tatwell I didn't realize, or maybe I knew at one time and have forgotten, that the velocity formats are in github. They can't run in the local environment and there is version control for these in Cascade so why do we keep them in github as well making this more work to update them multiple places? I don't think I have been updating github with format changes I made in recent months. I also discovered the data definitions are in there. I haven't been updating them there either. Same question.

File: .cascade-code/_cascade/formats/modular/widgets/Featured_News_Events_Left_Col (hunk 1 above)
Comment (tatwell) [reply .cascade-code/_cascade/formats/modular/widgets/Featured_News_Events_Left_Col]: New ones should. These are updates to existing one.

**Comment by tatwell** [comment]
@mandythomas The dev team just started putting formats in Github. It's the same practice as with data definitions -- except with these there is no attempt to make them functional within the Cascade Assets demos (why they're in a `.` directory).

There's a few advantages with tracking them in Github:

- Easier to share and review code (main advantage)
- With code reviews, easier to establish and promote best practices
- More full-featured version control (e.g. branching and tagging)

As with data definitions, the version in production should always be considered the master version. When devs work on a format in Github, they should always be copying the version from Cascade production as the first step. Working through this Github repo is not required, but I think it's a good idea.

**Pull request merged**